### PR TITLE
Fix Raleway font on local version of macOS

### PR DESCRIPTION
**Pull request by fabienrohrer**
- [x] Restore woff file.
- [x] Use a system font on the local version of macOS.

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: local_exporter.py [hunk 1]
```diff
@@ -87,6 +87,10 @@ def download(url, target_file_path):
 
 reg = r'"https://www\.cyberbotics\.com/([^"]*)"'
 content = re.sub(reg, r'"dependencies/\1"', content)
+if platform.system() == 'Darwin':
+    # Because of the bad support of @font-face by QtWebKit on macOS,
+    # a custom CSS containing a system CSS is preferred for macOS.
```
Comment (stefaniapedrazzi) [inline local_exporter.py]: system CSS -> system font?

**Comment by fabienrohrer** [comment]
👍

File: css/main.mac_os.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  font-family: Tahoma, Arial, sans-serif;
+  font-size: 14px;
+}
+@font-face {
+  font-family: "Raleway";
+  font-style: normal;
+  font-weight: 100;
+  /*
+    Issue: QtWebKit font-face support is buggy on macOS.
+    Workaround: Do not define a source a let a system font be loaded.
```
Comment (stefaniapedrazzi) [inline css/main.mac_os.css]: a let -> and let?

**Comment by fabienrohrer** [comment]
Thank you for the quick review.

👍

**Pull request merged**